packages/@uppy/companion/KUBERNETES.md

```diff
@@ -28,6 +28,7 @@ data:
   COMPANION_DOMAINS: 'sub1.domain.com,sub2.domain.com,sub3.domain.com'
   COMPANION_PROTOCOL: 'YOUR SERVER PROTOCOL'
   COMPANION_STREAMING_UPLOAD: true
+  COMPANION_DEFERRED_UPLOAD_LENGTH: true
```

Comment:
If it's only for tus we should name it accordingly
```suggestion
  COMPANION_TUS_DEFERRED_UPLOAD_LENGTH: true
```

Reply:
Good idea!

Reply:
Done